File: dailyQuestion3471.java
class Solution {
    public int largestInteger(int[] nums, int k) {

        int[] count = new int[51];

        for (int i = 0; i <= nums.length - k; i++) {

            boolean[] seen = new boolean[51];

            for (int j = i; j < i + k; j++) {

                int x = nums[j];

                if (!seen[x]) {
                    count[x]++;
                    seen[x] = true;
                }
            }
        }

        for (int x = 50; x >= 0; x--) {
            if (count[x] == 1) {
                return x;
            }
        }

        return -1;
    }
}

File: dailyQuestion3471.c
int largestInteger(int* nums, int numsSize, int k) {

    int sol[51] = {0};

    for (int i = 0; i <= numsSize - k; i++) {

        int seen[51] = {0};

        for (int j = i; j < i + k; j++) {

            int x = nums[j];

            if (seen[x] == 0) {
                sol[x]++;
                seen[x] = 1;
            }
        }
    }

    for (int i = 50; i >= 0; i--) {
        if (sol[i] == 1) {
            return i;
        }
    }

    return -1;
}